### ISSUE TITLE
Using NVDA to perform mouse clicks fails in Java Swing applications

### ISSUE THREAD

**Issue by zersiax** [nvaccess/nvda#12089]
### Steps to reproduce:
1. Open a random Java Swing application, I have reproduced this on various ones.
2.  object navigate to a control.
3. Use nvda+shift+m (laptop layout) to route the mouse to the object.
4. nvda+[ to left click.

### Actual behavior:
God knows what. It clicks, but it seems entirely random if it will click above, next to or below the object in question. It rarely actually clicks the object you want, though.
### Expected behavior:
The object in the object nav focus should receive the click event. The only way to get this to happen consistently is to use nvda+enter instead, but this has its own problems, discussed in another issue.

**Comment by zersiax** [comment]
Closing this. This has to do with Windows 10 setting scaling on certain displays to 150% as it considers this a recommended setting. Some apps apparently don't think that recommendation is warranted, which causes clicks to not properly target the right spot.
